janusgraph-core/src/main/java/org/janusgraph/graphdb/transaction/TransactionConfiguration.java

```diff
@@ -190,4 +190,10 @@ public interface TransactionConfiguration extends BaseTransactionConfig {
      */
     boolean hasRestrictedPartitions();
 
+    /**
+     * Returns true if read queries should skip accessing JanusGraph database level cache (db-cache).
+     * Doesn't have any effect if database level cache was disabled during JanusGraph instance creation.
```

Comment:
Same as above

Reply:
Thank you. Updated documentation as you suggested